wp/lib/bap_wp/src/environment.ml

```diff
@@ -60,7 +62,7 @@ type t = {
   indirect_handler : indirect_spec;
   jmp_handler : jmp_spec;
   int_handler : int_spec;
-  loop_handler : loop_handler;
+  loop_handler : loop_handle;
```

Comment:
Why did we change the name?

Reply:
I created a type that holds the different types of handlers we could use for loops. The names were an oversight on my part. I can change the names back.

https://github.com/draperlaboratory/cbat_tools/blob/993075cb5abb04473ac7b82c819cb485b7e694e8/wp/lib/bap_wp/src/environment.ml#L92

https://github.com/draperlaboratory/cbat_tools/blob/993075cb5abb04473ac7b82c819cb485b7e694e8/wp/lib/bap_wp/src/environment.ml#L94